edb/pgsql/resolver/command.py

```diff
@@ -696,13 +797,7 @@ def _uncompile_insert_pointer_stmt(
 def _has_at_most_one_row(query: pgast.Query | None) -> bool:
     if not query:
         return True
-    return isinstance(query, pgast.SelectStmt) and (
-        (query.values and len(query.values) <= 1)
-        or (
-            isinstance(query.limit_count, pgast.NumericConstant)
-            and query.limit_count.val == '1'
-        )
-    )
```

Comment:
This change should be reverted: it is here because otherwise the pgcompiler will produce other bugs.